doc/quick-start.rst

```diff
@@ -6,10 +6,148 @@ This document gives simple usage examples of Dune. You can also look at
 `examples <https://github.com/ocaml/dune/tree/master/example>`__ for complete
 examples of projects using Dune.
 
+Initializing projects
+=====================
+
+The following subsections illustrate basic usage of the ``dune init proj``
+subcommand. For more documentation see :ref:`initializing_components` and the
+inline help available from ``dune init --help``.
+
+.. _initializing-an-executable:
+
+Initializing an executable
+--------------------------
+
+To initialize a project to build an executable program, run the following
+(replacing ``project_name`` with the name of your project):
+
+.. code:: shell
+
+    dune init proj project_name
+
+This will create a directory for the project that includes the following contents:
```

Comment:
This creates a project directory that includes the following contents: 
...or...
This creates a directory for the project that includes the following contents: